pkg/sloop/webfiles/sloop_ui.js

```diff
@@ -43,35 +61,41 @@ let cmpFn = noSortFn;
 
 payload = d3.json(dataQueryUrl);
 payload.then(function (result) {
+    initializeDimensions();
     let svg = render(result);
     bindMouseEvents(svg);
     appendAxes(svg);
     renderTooltip();
 });
 
+function colorGen(scheme) {
+
+}
+
 function render(result) {
     let data = processAndSortResources(result);
     let dataByKind, kinds, filteredData;
 
+
     if (!data) {
         xAxisScale = d3.scaleUtc()
             .range([margin.left, displayMaxX - margin.left]);
 
+
         yAxisBand = d3.scaleBand()
-            .range([margin.top, (50) - margin.top])
-            .padding(0.2);
+            .range([margin.top, (yAxisOffset) - margin.top])
+            .padding(yAxisPadding);
 
 
-        axisTop = d3.axisTop(xAxisScale);
+        axisTop = d3.axisTop(xAxisScale)
         axisBottom = d3.axisBottom(xAxisScale);
         filteredData = []
     } else {
         dataByKind = d3.nest().key(d => d.kind).entries(data);
         kinds = dataByKind.map(d => d.key);
 
-        barColorGenFunc = d3.scaleOrdinal(d3.schemeSet2).domain(kinds);
-        severityColorGenFunc = d3.scaleLinear().domain([0, 1, 2]).range(['#4BD855', '#D8D14B', '#D84B4B']);
-        eventKindColorGenFunc = d3.scaleOrdinal(d3.schemeSet3).domain([]);
+        barColorGenFunc = d3.scaleOrdinal().domain(kinds).range(palette.highlight);
+        severityColorGenFunc = d3.scaleLinear().domain([0, 1, 2]).range(['#4BD855', '#E0E000', '#D84B4B']);
```

Comment:
should these colors be moved in to the palette?

Reply:
Yup